apps/dolly-frontend/src/main/js/src/components/bestillingsveileder/stegVelger/steg/steg1/paneler/Adresse.tsx

```diff
@@ -30,6 +31,16 @@ export const AdressePanel = ({ stateModifier, formValues }: any) => {
 				<Attributt attr={sm.attrs.kontaktadresse} />
 			</AttributtKategori>
 
+			<AttributtKategori title="Delt bosted" attr={sm.attrs}>
+				<Attributt
+					attr={sm.attrs.deltBosted}
+					infoTekst={
+						'Opplysningstypen “Delt bosted” legges til på barnet(barna), men kan bestilles på hovedperson/partner/barn. ' +
+						'Vilkåret er at barn(a) har to foreldre med forskjellig norsk bostedsadresse.  Master=FREG kun.'
```

Comment:
Mulig å skrive siste setning litt penere eller? "Gjelder kun for master FREG" eller noe i den duren? 🤔